src/app/clusters/service-area-server/README.md

```diff
@@ -0,0 +1,35 @@
+# The Service Area Cluster
+
+## Cluster design
+
+The service area cluster is designed around three core classes: `Instance`,
+`MemoryDelegate`, and a second `Delegate` class.
+
+The `Instance` class serves as the primary interface for safely accessing and
+modifying cluster attributes, adhering to the constraints outlined in the
+specification document. However, it does not handle the storage of list
+attributes.
+
+Storage management is delegated to the `MemoryDelegate` class, which provides a
```

Comment:
So should this thing be called StorageDelegate or StateDelegate?  MemoryDelegate is a bit odd...

Reply:
Renamed to `StorageDelegate`. Unfortunately I forgot to add some untracked files so this is across 2 commits.
- https://github.com/project-chip/connectedhomeip/pull/35176/commits/bebe55d7b8deb61c25fc9364cab36f50597d410a
- https://github.com/project-chip/connectedhomeip/pull/35176/commits/1820eb7569af48875b9eab1411ffcaf17f406808